app/src/main/java/com/kickstarter/extensions/ActivityExt.kt

```diff
@@ -0,0 +1,6 @@
+package com.kickstarter.extensions
+
+import android.app.Activity
+import com.kickstarter.R
+
+fun Activity.bottomSlideAnimation() = overridePendingTransition(R.anim.settings_bottom_slide, R.anim.fade_out)
```

Comment:
Let's get rid of this extension and use what already exists: `BaseActivity` has a method called `exitTransition` that you would override in `SettingsActivity`.

So instead of adding `bottomSlideAnimation` every time you start a new activity, you'd just add 

```  @Override
  protected @Nullable Pair<Integer, Integer> exitTransition() {
    return slideUpFromBottom();
  } 
```

where `slideUpFromBottom()` is a Pair of animation resource IDs defined in `TransitionUtils`:

```
  public static @NonNull Pair<Integer, Integer> slideUpFromBottom() {
    return Pair.create(R.anim. settings_bottom_slide, R.anim. fade_out);
  }
```